3-UsingAPIs/Week1/homework/ex4-pokerDiceAll.js

```diff
@@ -29,7 +29,8 @@ const rollDie = require('../../helpers/pokerDiceRoller');
 function rollDice() {
   // TODO Refactor this function
   const dice = [1, 2, 3, 4, 5];
-  return rollDie(1);
+  const promises = dice.map((die) => rollDie(die));
```

Comment:
You are passing a function to `dice.map()` that takes a number and returns a promise returned by `rollDie()`. But `rollDie()` itself also takes a number and returns the same promise. So no need for this intermediate (arrow) function:

```suggestion
  const promises = dice.map(rollDie);
```

You could even shorten it to:

```js
function rollDice() {
  const dice = [1, 2, 3, 4, 5];
  return Promise.all(dice.map(rollDie));
}
```